cmake/GinkgoConfig.cmake.in

```diff
@@ -36,6 +36,7 @@ set(GINKGO_BUILD_REFERENCE @GINKGO_BUILD_REFERENCE@)
 set(GINKGO_BUILD_OMP @GINKGO_BUILD_OMP@)
 set(GINKGO_BUILD_CUDA @GINKGO_BUILD_CUDA@)
 set(GINKGO_BUILD_HIP @GINKGO_BUILD_HIP@)
+set(GINKGO_BUILD_MPI @GINKGO_BUILD_MPI@)
```

Comment:
I think we need the HAVE_MPI because we want to include it in the config.hpp.in file, which is possible with HAVE_MPI because it is a boolean, while BUILD_MPI is not a boolean.

Reply:
CMake has your back :) You might be looking for `#cmakedefine01`? https://cmake.org/cmake/help/latest/command/configure_file.html

Reply:
Nice. I did not know about `#cmakedefine01`. I guess we could also update the HWLOC and PAPI with those now.

Reply:
Probably. Also, you might not need to propagate the HAVE_* flags inside GinkgoConfig.cmake at all if you don't need to use them inside CMake, since they are already defined in the exported config.hpp.